rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
```

Comment:
- [ ] The rental has to add itself to the @person to achieve the `belongs-to association` like so `person.rentals << self`, please kindly fix that.

Reply:
You have implemented it well. :heavy_check_mark: